app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -1166,16 +1169,6 @@ class ChatRoomPresenter @Inject constructor(
         }
     }
 
-    private fun logMessageSent() {
-        when {
-            roomTypeOf(chatRoomType) is RoomType.DirectMessage ->
-                analyticsManager.logMessageSent(SubscriptionTypeEvent.DirectMessage)
-            roomTypeOf(chatRoomType) is RoomType.Channel ->
-                analyticsManager.logMessageSent(SubscriptionTypeEvent.Channel)
-            else -> analyticsManager.logMessageSent(SubscriptionTypeEvent.Group)
-        }
-    }
-
```

Comment:
currently Subscription Type is something that's tracked in firebase and seems like something we would want in the future. Am I right in saying we would no longer be tracking that? or am I misunderstanding how that will work with message_type?

Reply:
Its just a string. Kind of a dumb over-engineering thing in my opinion.